data.json

```diff
@@ -1306,5 +1306,12 @@
     "githubUsername": "BADSHAH-AMAN",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/132057221-2e7c6dc4-a664-4565-9909-101b8fadfa4b.png",
     "issueNumber": 685
-  }
+  },
+  
```

Comment:
```suggestion
```

Reply:
I don't understand. can your please explain, should I remove red line.